### ISSUE TITLE
pts/enconde-mp3 -- Fixing configuration file shows up to 4x improvement in POWER9 system

### ISSUE THREAD

**Issue by johnjmar** [phoronix-test-suite/test-profiles#34]
Before running `autoconf`, I changed the following in the configure.in file:

```
@@ -94,9 +94,9 @@
 
 	if test "${HAVE_GCC}" = "yes"; then
 		AC_MSG_CHECKING(version of GCC)
-		GCC_version="`${CC} --version | sed -n '1s/^[[^ ]]* (.*) //;s/ .*$//;1p'`"
+		GCC_version="`${CC} --version | head -n1 | awk '{print $3}' `"
 		case "${GCC_version}" in 
-		[0-9]*[0-9]*)
+		[[0-9]]*[[0-9]]*)
 			AC_MSG_RESULT(${GCC_version})
 			;;
 		*)
@@ -701,7 +701,7 @@
 	AC_DEFINE(TAKEHIRO_IEEE754_HACK, 1, IEEE754 compatible machine)
 	AC_DEFINE(USE_FAST_LOG, 1, faster log implementation with less but enough precission)
 	;;
-powerpc)
+powerpc*|ppc64*)
 	CPUTYPE="no"
 
 	# use internal knowledge of the IEEE 754 layout
```

**Comment by johnjmar** [comment]
Some details:
```
Phoronix Test Suite v8.2.0m3

    To Install:    pts/timed-audio-encode-1.0.0
    To Install:    pts/encode-mp3-1.7.0        

    Determining File Requirements ....................................................................................................
    Searching Download Caches ........................................................................................................

    2 Tests To Install
        5MB Of Disk Space Is Needed

    pts/timed-audio-encode-1.0.0:
        Test Installation 1 of 2 
        1 File Needed [75 MB]    
        File Found: pts-trondheim-wav-3.tar.gz                                                                               [75.00MB]
        Installing Test @ 16:23:42                                                                                                    

    pts/encode-mp3-1.7.0:
        Test Installation 2 of 2
        1 File Needed [1.45 MB] 
        File Found: lame-3.100.tar.gz                                                                                         [1.45MB]
        Installation Size: 5 MB                                                                                                       
        Installing Test @ 16:23:45                                                                                                    

System Information


  PROCESSOR:          POWER9 altivec supported @ 3.80GHz
    Core Count:       160                               
    Scaling Driver:   powernv-cpufreq performance       

  GRAPHICS:           LLVMpipe
    OpenGL:           1.4 (3.0 Mesa 17.0.6) Gallium 0.4 (LLVM 3.8 256 bits)
    Screen:           1440x900                                             

  MOTHERBOARD:        PowerNV 8335-GTH
    Network:          Broadcom Limited NetXtreme BCM5719 Gigabit PCIe

  MEMORY:             584704MB

  DISK:               2 x 1920GB MTFDDAK1T9TCB
    File-System:      xfs                     
    Mount Options:    attr2 inode64 noquota relatime rw seclabel
    Disk Scheduler:   DEADLINE                                  

  OPERATING SYSTEM:   Red Hat Enterprise Linux 7.5
    Kernel:           4.14.0-49.10.1.el7a.ppc64le (ppc64le)
    Compiler:         GCC 4.8.5 20150623 + CUDA 9.2        
    Security:         SELinux + Indirect branch serialisation (kernel only) Indirect branch cache disabled ori31 speculation barrier enabled Protection                                                                                                                             
```

**Comment by johnjmar** [comment]
```
=== VANILLA COMPILATION ===
LAME MP3 Encoding 3.100:
    pts/encode-mp3-1.7.0                                                                                                                  
    Test 1 of 1                                                                                                                           
    Estimated Trial Run Count:    3                                                                                                       
    Estimated Time To Completion: 4 Minutes [16:27 CDT]                                                                                   
        Started Run 1 @ 16:24:00                                                                                                          
        Started Run 2 @ 16:25:16                                                                                                          
        Started Run 3 @ 16:26:32                                                                                                          

    WAV To MP3:
        75.110855817795
        75.125329017639
        75.107865095139

    Average: 75.11 Seconds
    Deviation: 0.01% 

=== OPTIMIZED COMPILATION ===
LAME MP3 Encoding 3.100:
    local/encode-mp3-1.7.0                                                                                                                
    Test 1 of 1                                                                                                                           
    Estimated Trial Run Count:    3                                                                                                       
    Estimated Time To Completion: 2 Minutes [16:32 CDT]                                                                                   
        Started Run 1 @ 16:30:37                                                                                                          
        Started Run 2 @ 16:30:55                                                                                                          
        Started Run 3 @ 16:31:14                                                                                                          

    WAV To MP3:
        17.847348928452
        17.854550123215
        17.8644490242  

    Average: 17.86 Seconds
    Deviation: 0.05%    
```

**Comment by michaellarabel** [comment]
Apologies for the delay, the patch should now be included as part of the pts/encode-mp3-1.7.1 test profile update.
